test/WebJobs.Script.Tests.Shared/TestHelpers.Constants.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests
+{
+    public static partial class TestHelpers
+    {
+#if DEBUG
+        public const string BuildConfig = "debug";
+#else
+        public const string BuildConfig = "release";
+#endif
+        // Not a real storage account key.
+        public static readonly string StorageAccountKey = Convert.ToBase64String(Encoding.UTF8.GetBytes("PLACEHOLDER"));
```

Comment:
What part of this change will cause the detector to not flag it? Changing a "secret looking" string to a non secret looking string in the code ?

Reply:
"PLACEHOLDER" is a known value to credscan and it will recognize that as not a real secret.